ephios/plugins/basesignup/signup/common.py

```diff
@@ -102,3 +103,36 @@ def get_configuration_fields(self):
                 },
             }
         )
+
+    def get_participation_display(self):
+        relevant_qualification_categories = global_preferences_registry.manager()[
+            "general__relevant_qualification_categories"
+        ]
+        participation_info = [
+            [
+                f"{participant.first_name} {participant.last_name}",
+                ", ".join(
+                    participant.qualifications.filter(
+                        category__in=relevant_qualification_categories
+                    ).values_list("title", flat=True)
+                ),
+            ]
+            for participant in self.shift.get_participants()
+        ]
+        try:
+            # we may not be using MinMaxParticipantsMixin so there may be no bounds
```

Comment:
Well that comment is not completely right. The Mixin only provides one way to implement `get_participant_count_bounds`. SectionBasedSignup doesn't use it and still implements the method.

Reply:
And plugins might use the QualificationMixin but not the MinMaxMixin but their own way...